qiskit/test/mock/utils/configurable_backend_v2.py

```diff
@@ -0,0 +1,183 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Configurable backend for BackendV2."""
+import datetime
+from typing import Optional, List, Tuple, Union, Dict
+
+import numpy as np
+
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.parameter import Parameter
+from qiskit.circuit.measure import Measure
+from qiskit.circuit.reset import Reset
+from qiskit.providers.backend import BackendV2, QubitProperties
+from qiskit.providers.options import Options
+from qiskit.transpiler import Target, InstructionProperties
+
+
+class ConfigurableFakeBackendV2(BackendV2):
+    """Factory for generating mock backendv2 instances"""
+
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        n_qubits: int,
+        gate_configuration: Dict[Gate, List[Tuple[int]]],
+        measurable_qubits: List[int] = None,
+        parameterized_gates: Optional[Dict[Gate, str]] = None,
+        qubit_coordinates: Optional[List[List[int]]] = None,
+        qubit_t1: Optional[Union[float, List[float]]] = None,
+        qubit_t2: Optional[Union[float, List[float]]] = None,
+        qubit_frequency: Optional[Union[float, List[float]]] = None,
+        qubit_readout_error: Optional[Union[float, List[float]]] = None,
+        single_qubit_gates: Optional[List[str]] = None,
```

Comment:
How is this parameter used? It doesn't seem to match up with `gate_configuration` or `parameterized_gates`. For example, you could specify completely different gates here and there would be nothing that errors. I'm just wondering if we can remove this argument?

Reply:
Looks like this was accidentally left in from an old version - can be removed.